src/stores/user-store.js

```diff
@@ -66,7 +85,17 @@ class UserStore extends EventEmitter {
         }
         else if (action.ACTION_TYPE === ACTION_TYPE.updateUser) {
             this._updateUser(action.user);
-            this.emit(EVENT_TYPE.change);
+            if (action.searchMode) {
+                const _foundUsers = this._findUser(this._searchedUser);
+                this.emit(EVENT_TYPE.usersFound, _foundUsers, this._defineSearchFields());
+            } else {
+                this.emit(EVENT_TYPE.change);
+            }
+        }
+        else if (action.ACTION_TYPE === ACTION_TYPE.findUser) {
+            this._searchedUser = action.user;
```

Comment:
it's better to have one handler metod for each action than have inline logic in one `registerActions` method